src/components/shared/card/Card.tsx

```diff
@@ -0,0 +1,36 @@
+import { BsDot } from "react-icons/bs";
+import { useNavigate } from "react-router-dom";
+
+export type CardProps = {
+  id: string;
+  titleName: string;
+  description: string;
+  imgSrc: string;
+};
+
+export function Card({ titleName, description, imgSrc, id }: CardProps) {
+  const navigate = useNavigate();
+
+  const handleClick = () => {
+    navigate(`/doctor/${id}`);
+  };
+
+  return (
+    <div
+      onClick={handleClick}
+      className="bg-white w-52 h-80 rounded-xl flex flex-col items-center p-4 cursor-pointer shadow-md hover:shadow-xl transition-shadow duration-300 border border-gray-200"
+    >
+      <img src={imgSrc} alt="Doctor" className="w-28" />
+
+      <div className="flex items-center mb-2">
+        <BsDot className="text-green-500" />
+        <span className="text-green-500 text-sm font-medium">Available</span>
+      </div>
+
+      <h1 className="text-lg font-semibold text-gray-800 mb-1 text-center">
+        {titleName}
+      </h1>
+      <h2 className="text-sm text-gray-500 text-center">{description}</h2>
+    </div>
```

Comment:
Nie lepiej opakować całą kartę Linkiem, niż robić diva który po kliknięciu gdzieś nas przeniesie? Prawdopodobnie nie możemy tego linku teraz wywołać tabulatorem i enterem, bo jest to div, a zwykły link moze byc zaznaczany za pomocą taba klawiaturą

Reply:
+ będzie miał focus